src/module/token/token-hud.js

```diff
@@ -0,0 +1,124 @@
+export class SFRPGTokenHUD extends TokenHUD {
+
+    /**
+     * @override
+     * Calls setCondition from the SFRPG actor before calling super
+     * @param {Event} event 
+     * @param {any} param1 - contains overlay boolean
+     * @returns super
```

Comment:
Looks like it returns a boolean to me, and super is not called anywhere?

Reply:
Yep, refactored out to fix some bullshit